connaisseur/tests/integration/LOCAL.md

```diff
@@ -0,0 +1,37 @@
+# Local integration test setup
+
+## Minikube
+
+Assuming you have a running minikube cluster set as current kubernetes context you just need to have the alerting interface running.
+
+Run the docker container that serves as alerting endpoint and retrieve the IP address that is has on the `bridge` network:
+
+```
+docker run -d --name alerting-endpoint -p 56243:56243 docker.io/securesystemsengineering/alerting-endpoint:latest
```

Comment:
Dis don wörk wiff ma set app, becose assa netwörk

Reply:
Differentiated between virtualbox and docker driver for minikube now :) Thanks for the hint!